contrib/docker/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM mcr.microsoft.com/playwright:v1.44.1
+# The chromium version must be updated with playwright container upgrade
+ENV CHROME_PATH /ms-playwright/chromium-1117/chrome-linux/chrome
+
+# Required to run as pwuser chromium sandbox security
+USER pwuser
+# Add package.json for each workspace
+WORKDIR /app/elements/openlayers-core
+COPY --chown=pwuser elements/openlayers-core/package.json .
+WORKDIR /app/elements/openlayers-elements
+COPY --chown=pwuser elements/openlayers-elements/package.json .
+WORKDIR /app/elements/swisstopo-elements
+COPY --chown=pwuser elements/swisstopo-elements/package.json .
+WORKDIR /app
+COPY --chown=pwuser package.json package-lock.json .
+# Workaround for MacOS issues
+VOLUME /app/node_modules
```

Comment:
```suggestion
```

This appears unnecessary after all. When I first ran this I got another `ENOENT` error about npm logs directory. I tried some things and then it would stop working altogether and just fail with `mkdir: cannot create directory ‘/app/node_modules’: No space left on device` 😵‍💫

After `docker system prune` it simply worked 😄